### PR TITLE
toggle all elements

### LINKED ISSUE #4: Add a Select All button
Which will toggle selection

### PR DESCRIPTION
**Pull request by Johann-S**
Fixes #4

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ $(() => {
   createModal()
 
   const $form = $('form')
-  const $checkBoxRequirePopper = $('.require-popper')
+  const $checkBoxRequirePopper = $(document.querySelector('.require-popper'))
```
Comment (MartijnCuppens) [inline src/index.js]: What's this for?

**Comment by MartijnCuppens** [comment]
Only the checkboxes of the first group are prefixed with `.js-`, shouldn't we do this with all elements?

Maybe we can also do something like:
```html
<button class="btn btn-outline-secondary btn-sm float-right js-btn-toggle" data-target=".js-javascript-checkbox">
    Toggle all
</button>
and
<button class="btn btn-outline-secondary btn-sm float-right js-btn-toggle" data-target=".js-layout-checkbox">
    Toggle all
</button>
```

```js
$('.js-btn-toggle').on('click', function (event) {
  event.preventDefault()
  const $targets = $($(this).data('target'));
  // ...
})
```

so that we don't need the `if/else` construction?

File: src/index.js (hunk 1 above)
Comment (Johann-S) [reply src/index.js]: Just a matter of perf, that's always better to query our elements ourselves instead of using jQuery

**Comment by Johann-S** [comment]
I did that because it was for the JS part (`.js-checkbox` because we are in the JavaScript plugin section)

And with your solution, that's right we don't have `if/else` but we query our elements on each click, instead of one

File: src/index.js (hunk 1 above)
Comment (MartijnCuppens) [reply src/index.js]: If we're going to optimise this here, we better do this everywhere. But this is out of the scope of this PR I think.

**Comment by MartijnCuppens** [comment]
@Johann-S, maybe there is a solution which includes the best parts of both solutions?

**Comment by Johann-S** [comment]
I don't know 🤔 

I'll merge this PR like that and maybe we'll change that later if we have ideas

**Pull request merged**